index.html

```diff
@@ -291,7 +386,14 @@ <h3 id="ruby-character-size-and-character-placement">Ruby character size and cha
 <section>
 <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
 
-<p>Mono-ruby is placed as follows:</p>
+<p>Mono-ruby is placed as follows. 
+<span class="checkme">
+To align following items to the two step processing method described in 
```

Comment:
I think we can add an id to the `li` element of the two-step processing method and link to it directly.

Reply:
added to #22